packages/frontend/src/components/PieChartConfig/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button, Popover } from '@mantine/core';
+import { IconCaretDown } from '@tabler/icons-react';
+import React from 'react';
+import MantineIcon from '../common/MantineIcon';
+import PieLayoutConfig from './PieChartLayoutConfig';
+
+const PieChartConfig: React.FC = () => {
+    return (
+        <Popover position="bottom" shadow="xl" withArrow keepMounted={false}>
+            <Popover.Target>
+                <Button
+                    variant="subtle"
```

Comment:
I think the variant `subtle` makes it look _disabled_ next to the others...
<img width="567" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/679b27f0-0306-45cb-b6de-df77ff30a683">

Reply:
Also, when I had this open, and then clicked on `filters`, it remained open...
<img width="689" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/7e4e02de-1820-47f3-a4db-20c349d1e7e1">

Reply:
- working on it with @PriPatel to find a proper variants.
- re: popover remains open: this is fixed here #6045